projects/ion/src/lib/picker/date-picker/date-picker-input/date-picker-input.component.html

```diff
@@ -0,0 +1,11 @@
+<div class="container-input" data-testid="container-input">
+  <ion-input
+    [value]="date"
+    [placeholder]="placeholder"
+    [iconInput]="date ? '' : 'calendar'"
+    iconDirection="right"
+    [inputIconButton]="!date ? false : true"
```

Comment:
```suggestion
    [inputIconButton]="!!date"
```
It could be simplified to this? I'm not sure.